src/libs/actions/Policy.js

```diff
@@ -217,6 +240,16 @@ function removeMembers(accountIDs, policyID) {
             key: membersListKey,
             value: _.object(accountIDs, Array(accountIDs.length).fill({errors: ErrorUtils.getMicroSecondOnyxError('workspace.people.error.genericRemove')})),
         },
+        ..._.map(workspaceChats, ({reportID, stateNum, statusNum, hasDraft, oldPolicyName}) => ({
```

Comment:
We should also remove the closed report actions if we failed by setting them to null